language/stdlib/modules/sliding_nonce.move

```diff
@@ -0,0 +1,45 @@
+address 0x0 {
+module SlidingNonce {
+    use 0x0::Transaction;
+
+    resource struct T {
+        min_nonce: u64,
+        nonce_mask: u128,
+    }
+
+    // Aborts transaction if nonce can not be accepted
+    public fun accept_nonce(seq_nonce: u64) acquires T {
+        if(seq_nonce == 0) {
```

Comment:
Can you explain the motivation behind this a bit more? It seems like a potential invitation to replays if an approval gets tagged with nonce 0.

Reply:
Nit: space between `if` and `(` (and same below). Sorry, we don't have an autoformatter yet...